src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CountFalsePositives.java

```diff
@@ -32,62 +30,101 @@
 /**
  * Created by tsato on 12/28/16.
  */
-public class CountFalsePositives extends CommandLineProgram {
-    @Argument(doc = "a line-delimited list of mutect2 vcf files", fullName= "", shortName = "V", optional = false)
-    protected File vcfList;
 
-    @Argument(doc = "output file", fullName= "output", shortName = "O", optional = false)
-    protected File output;
+/** TODO: this class should be a variant walker. That way we can ask questions like how are the false positive rates
+ *  over these intervals and so on. For now, we look at the entire vcf and assume ICE intervals
+ */
+
+public class CountFalsePositives extends VariantWalker {
+    @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME, shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME, doc = "Output file (if not provided, defaults to STDOUT)", common = false, optional = true)
```

Comment:
I just followed the convention in `ExampleVairntWalker.java` (if output is not given just print to stdout). But I'm ok with either approach.